### PR TITLE
Dpl 2107 upversion daap update schema

### PR DESCRIPTION
**Pull request by LavMatt**
collect and return response from the copy functionality added, where a major version increment is inferred from update.

Also return more information re. increment type in response.

### CODE REVIEW

File: containers/daap-update-schema/src/var/task/update_schema.py [hunk 1]
```diff
@@ -19,12 +19,14 @@ def handler(event, context):
...
         data_product_name=data_product_name,
         table_name=table_name,
         logger=logger,
         input_data=None,
-    ).exists
+    )
+
+    schema_exists = schema.exists
 
     if not schema_exists:
```
Comment (tom-webber) [inline containers/daap-update-schema/src/var/task/update_schema.py]:
```suggestion
    if not schema.exists:
```

File: containers/daap-update-schema/src/var/task/update_schema.py [hunk 2]
```diff
@@ -56,6 +60,18 @@ def handler(event, context):
...
-    return format_response_json(
-        status_code=HTTPStatus.OK, body={"version": new_version, "changes": changes}
-    )
+    # check if major or minor
+    if not int(new_version.split(".")[0][-1]) == int(schema.version.split(".")[0][-1]):
+        body_dict = {
+            "version": new_version,
+            "increment_type": "major",
+            "changes": changes,
+            **copy_response,
```
Comment (tom-webber) [inline containers/daap-update-schema/src/var/task/update_schema.py:67]: Should we change `VersionManager.update_schema` to return this dictionary, given all of this information is handled as part of the `update_schema` function anyway?

Comment (LavMatt) [reply containers/daap-update-schema/src/var/task/update_schema.py:67]: Yeah possibly. My thoughts on it were that it should be done as a wider review/refactor of all responses. I think we'll need to ensure better consistency as we've been doing our own and not following a strict framework (i haven't at least...), other than to be informative/useful responses.

**Review by tom-webber — APPROVED** [review]
LGTM

**Pull request merged**